imu_merger:
  ros__parameters:
    frame_id: "base_link"
    orientation_covariance: [0.01, 0.0, 0.0,
                               0.0, 0.01, 0.0,
                               0.0, 0.0, 0.01]
    angular_velocity_covariance: [0.1, 0.0, 0.0,
                                  0.0, 0.1, 0.0,
                                  0.0, 0.0, 0.1]
    linear_acceleration_covariance: [0.02, 0.0, 0.0,
                                     0.0, 0.02, 0.0,
                                     0.0, 0.0, 0.02]